tests/unit/amo/components/TestAddonMeta.js

```diff
@@ -99,30 +99,30 @@ describe(__filename, () => {
     it('renders a count of multiple ratings', () => {
       const root = renderRatings({ count: 5 });
 
-      expect(getReviewCount(root).content).toEqual('5');
-      expect(getReviewCount(root).title).toEqual('Ratings');
+      expect(getReviewCount(root).content.props.children).toEqual('5');
```

Comment:
Because the component has been changed to send `Link`s to the `MetadataCard` that is what we should be checking. We should check that the property of the `MetadataCard` is the `Link` that we expect it to be. This is true for all of the tests that check anything that has been set to a `Link`.

Reply:
Here's an example of how you can get at the list of links that were passed into the `MetadataCard` and do some assertions about them:

```js
    it('renders links for each rating item', () => {
      const root = renderRatings({ count: 5 });
      const links = root.find(MetadataCard).dive().find(Link);

      expect(links.at(0)).toHaveProp('to', `/addon/${fakeAddon.slug}/reviews/`);
      expect(links.at(0).children()).toHaveText('5');
     ...etc.
    });
```

Where I've checked the first link, via `links.at(0)`, you could do the same for all 4 expected `Link`s.

Reply:
I see you added a test that checks the `Link` components below, but I think that logic should replace what you have above.